docs/dev/tutorials/cross-chain-tutorial.md

```diff
@@ -155,14 +155,20 @@ main().catch((error) => {
 
 5. Compile the contract and run the deployment script with:
 
-```
-# compile contract
+::: code-tabs
+
+@tab:active Compile contract
```

Comment:
please check the whole page and refactor all snippets with yarn, sh/bash etc
for example here yarn commands are still in text, is  some other places they are already in *yarn* tab active